src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_UsingStatement.cs

```diff
@@ -341,10 +341,13 @@ private BoundStatement RewriteUsingStatementTryFinally(SyntaxNode syntax, BoundB
                     hasErrors: false);
             }
 
+            // Terminate the try block with a hidden sequence point
+            BoundBlock terminatedTryBlock = tryBlock.WithStatements(tryBlock.Statements.Add(new BoundSequencePoint(null, null)));
```

Comment:
Ignore that. When I try those scenarios in the debugger, they are actually fine.